services/app-api/src/emailer/templateHelpers.ts

```diff
@@ -68,70 +68,69 @@ const includesChipPrograms = (programIDs: string[]): boolean => {
+    - cmsReviewSharedEmails added to all submissions by default
+    - dmcpEmails added in both CONTRACT_ONLY and CONTRACT_AND_RATES
+    - oactEmails added for CONTRACT_AND_RATES
+    
+    Return should be wrapped in pruneDuplicate to ensure even if config is added twice, we get unique list of reviewers
+*/
 const generateCMSReviewerEmails = (
     config: EmailConfiguration,
     pkg: LockedHealthPlanFormDataType | UnlockedHealthPlanFormDataType,
     stateAnalystsEmails: StateAnalystsEmails
 ): string[] | Error => {
-    // OACT email, receives only CONTRACT_AND_RATES submission, excludes CHIP and state of PR
-    const oactEmails: string[] = config.ratesReviewSharedEmails
-
-    // DMCP and OACT share an inbox that receives all submissions, excludes CHIP and state of PR and should be included
-    // in cmsReviewSharedEmails.
-    // cmsRateHelpEmailAddress is also set to this shared DMCP and OACT inbox. We are setting this const in order to
-    // filter it out for CHIP and State of PR submissions because we cannot store the actual email in the API. This
-    // will eventually be fixed when moving to storing emails in our DB.
-    const dmcpOACTSharedEmail: string = config.cmsRateHelpEmailAddress
-
-    // CHIP programs and state of PR submission does not include OACT and DMCP emails.
-    const filterChipAndPRSubmission = (reviewers: string[]) =>
-        reviewers.filter(
-            (email) =>
-                email !== dmcpOACTSharedEmail && !oactEmails.includes(email)
+    if (
+        pkg.submissionType !== 'CONTRACT_AND_RATES' &&
+        pkg.submissionType !== 'CONTRACT_ONLY'
+    ) {
+        return new Error(
+            `generateCMSReviewerEmails does not currently support submission type: ${pkg.submissionType}.`
         )
+    }
 
-    //This list of program IDs combines programs specified for contract certification and rate certification.
+    const { oactEmails, dmcpEmails, dmcoEmails } = config
     const programIDs = findAllPackageProgramIds(pkg)
+    let reviewers: string[] = []
 
-    // dmcpEmail is added in both CONTRACT_ONLY and CONTRACT_AND_RATES, in case it wasn't included in cmsReviewSharedEmails.
-    // cmsReviewSharedEmail emails are pulled from parameter store /configuration/email/reviewTeamAddresses.
-    // pruneDuplicateEmails function will remove dmcpEmail if it's a duplicate.
     if (pkg.submissionType === 'CONTRACT_ONLY') {
         // Contract submissions reviewer emails
-        let reviewers = pruneDuplicateEmails([
+        reviewers = [
             ...config.cmsReviewSharedEmails,
             ...stateAnalystsEmails,
-            dmcpOACTSharedEmail,
-        ])
-
-        //Remove OACT and DMCP emails from CHIP or State of PR submissions
-        if (includesChipPrograms(programIDs) || pkg.stateCode === 'PR') {
-            reviewers = filterChipAndPRSubmission(reviewers)
-        }
-
-        return reviewers
+            ...dmcoEmails,
+            ...dmcpEmails,
```

Comment:
This refactor is awesome! Much easier to get your head around who the reviewers are. 

I saw you updated the `oact` store email in dev and val to `"OACT Dev1" <mc-review-qa+OACT+dev1@truss.works>,"OACT Dev2" <mc-review-qa+OACT+dev2@truss.works>` where the division name is in the alias. Could you do this for the other divisions? It would make reviewing test emails easier.

Reply:
This is done!